extensions/wikia/Chat2/Chat.class.php

```diff
@@ -69,26 +72,34 @@ static public function banUser($userNameToKickBan, $kickingUser, $time, $reason)
 			'reason' => $reason,
 		] );
 
-		global $wgCityId;
-		wfProfileIn( __METHOD__ );
-		$errorMsg = "";
-		$PERMISSION_TO_KICKBAN = "chatmoderator";
-		$userToKickBan = User::newFromName($userNameToKickBan);
+		$kickBanPermission = 'chatmoderator';
```

Comment:
It would as its loaded into another VARIABLE_IN_ALL_CAPS later on as well, then used free form in several other places.  I had to pick some place to stop improving though.  There's already a lot of clean up changes for this P2! :/